keywords/keywords.robot

```diff
@@ -49,4 +49,13 @@ Email And Password Warning Should Appear
     Page Should Contain Element         ${EMAIL_WARNING}   
     Wait Until Element Is Visible       ${PASSWORD_WARNING}
     Page Should Contain Element         ${PASSWORD_WARNING} 
-    
+
```

Comment:
you did not put an underscore in your variable name. Should be ${tab_name} and not ${tab name}